lib/Controller/RoomController.php

```diff
@@ -769,7 +769,10 @@ public function addParticipantToRoom(string $token, string $newParticipant): Dat
 
 		$newUser = $this->userManager->get($newParticipant);
 		if (!$newUser instanceof IUser) {
-			return new DataResponse([], Http::STATUS_NOT_FOUND);
+			$newUser = $this->groupManager->get($newParticipant);
```

Comment:
this needs to be explicit, if you selected the group, the user with the same name should not be added.
Similar to when a room is created, the share tpye should be handed in and then reacted upon respectivly.